tests/functional/all_daemons/ceph-override.json

```diff
@@ -1,21 +1,26 @@
 {
-	"ceph_conf_overrides": {
-		"global": {
-			"osd_pool_default_pg_num": 12,
-			"osd_pool_default_size": 1
-		}
-	},
-	"cephfs_pools": [
-		{
-		  "name": "cephfs_metadata",
-		  "pgs": 8,
-		  "size": 1
-		},
-		{
-		  "name": "cephfs_data",
-		  "pgs": 8,
-		  "size": 1
-		}
-	],
+  "ceph_conf_overrides": {
```

Comment:
could you add `--extra-vars @ceph-override.json` in tox.ini (L64)?

thanks maggle!